DESCRIPTION

```diff
@@ -38,13 +38,13 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.0
 Depends: R (>= 3.5)
 Imports:
-    admiral,
+    admiral (>= 0.8.0),
     admiraldev,
-    dplyr,
-    lubridate,
-    magrittr,
-    rlang,
-    tidyselect
+    dplyr (>= 0.8.4),
+    lubridate (>= 1.7.4),
+    magrittr (>= 1.5),
+    rlang (>= 0.4.4),
+    tidyselect (>= 1.0.0)
```

Comment:
I am sure these are all fine @bundfussr, but for completeness, where are the version minimums that we need documented? As I see there are many later versions of each of these packages on CRAN.

Reply:
@sgorm123 , I am not sure what you mean by "documented".

Reply:
i would always look to the description file for these, so for me this PR itself covers this, and no further documentation needed.

Reply:
As discussed @bundfussr, I just wondered if there was maybe documentation/process in which minimum versions are reviewed and updated.  If the PR itself covers this, as Ross says, then no problem.  Thank you both for responding.